assets/js/components/ParticipantsList.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import {
+  SimpleGrid,
+  Box,
+  Table,
+  Flex,
+  Heading,
+  ButtonGroup,
+  Button,
+  Spacer,
+  Avatar,
+  Menu,
+  IconButton,
+  MenuButton,
+  MenuList,
+  MenuItem,
+} from "@chakra-ui/react";
+
+const ParticipantMenu = (props) => {
+  return (
+    <Menu>
+      <MenuButton as={IconButton} aroa-label="Options" icon={"Menu Icon"} variant="outline" />
```

Comment:
I would change the color of this button to make it easier to find.

Reply:
I changed the color but everything style-wise is temporary anyway